src/bundle/Resources/views/themes/admin/ui/component/input_date_and_time.html.twig

```diff
@@ -0,0 +1,36 @@
+{% import '@ibexadesign/ui/component/macros.html.twig' as html %}
+
+{% set is_small = is_small|default(false) %}
+{% set inline_flatpickr = inline_flatpickr|default(false) %}
+
+{% set wrapper_attr = wrapper_attr|default({})|merge({
```

Comment:
Not exactly, I'm using here our input_text component, which uses `wrapper_attr` https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/views/themes/admin/ui/component/input_text.html.twig#L4 so without changing it there (or adding extra div here, but it's pointless IMO) it can't be changed to `attr`